### ISSUE TITLE
Change timzeone for Squeezebox Radio

### ISSUE THREAD

**Issue by SvenSchiffner** [music-assistant/support#2633]
### What version of Music Assistant has the issue?

2.0.7

### What version of the Home Assistant Integration have you got installed?

2024.7.1.b0

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

Hello,
thank you for the great software. I used the LMS previously and switched to Music Assistant yesterday. It works great, but I found an issue. I'm living in Germany. So the correct timezone is UTC+2 in the summer. But my Squeezebox Radio showing the UTC time. I found no options to set the timezone in MASS. In HA I set the correct timezone.
I also looked in the source code. But I'm unfamiliar with slimproto and found no real hint, how the time is sent. Also in the menu of the Squeezebox Radios I found no option.

### How to reproduce

Use a Squeezebox Radio in another timezone.

### Music Providers

Independent

### Player Providers

Slimproto

### What version of Home Assistant Core are your running

2024.7.2

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by OzGav** [comment]
Did the time show correctly with LMS (or not show at all?)

**Comment by SvenSchiffner** [comment]
It showed correctly. But in the LMS you also had an option to set the language and the timezone. The MASS integration is also using my language settings from HA correctly and is displayed in German.

**Comment by SvenSchiffner** [comment]
I found to interesting function in the aioslimproto source code. [This ](https://github.com/home-assistant-libs/aioslimproto/blob/main/aioslimproto/client.py#L545) should be for older players like the boom, which gets the display data over slimproto and [this ](https://github.com/home-assistant-libs/aioslimproto/blob/main/aioslimproto/cli.py#L1132) should be the relevant function for newer players, which use the CLI. Their is no custom timezone. So in my opinion the timezone of the system is used. In the code for the HA add-on I found no hint for a custom timezone or the TZ environment variable. So maybe, the never player never got the correct local time.

**Comment by cociweb** [comment]
Are the timestamps OK in the downloadable log file? (Settings>Core)

**Comment by SvenSchiffner** [comment]
Good point, the timestamps in the logs are in my local time and not UTC.

**Comment by marcelveldt** [comment]
Not stale

**Comment by marcelveldt** [comment]
Head-up: We are updating our docs regarding the Slimproto provider. 
We will only support squeezelite based players from now on and support for any actual hardware devices is provided without guarantee or support. Plain simple reason for this is that we do not have the time or capacity to support all device specific quirks of the various squeezebox products (you already have a great product for that, its called LMS) and we're talking about 20+ years old hardware here. Even my last physical SQ hardware device now died.

We will accept PR's on the aioslimproto package and the slimproto provider to fix any compatibility issues but SB hardware devices will no longer be the focus. I will further clarify this by renaming the provider.

One idea that popped in my head is to add a LMS player provider, so people can use a intermediate LMS server in between to actually talk to the squeezebox hardware while MA will provide the media.
